Dockerfile

```diff
@@ -13,6 +13,9 @@ ADD . /vpt
 RUN git describe --always > VERSION
 
 RUN gem install bundler
-RUN bundle install --jobs $(nproc) --without development test --path vendor/bundle --deployment
+RUN bundle config set deployment 'true'
+RUN bundle config set path 'vendor/bundle'
+RUN bundle config set without 'development test'
+RUN bundle install --jobs $(nproc)
```

Comment:
doesn't this create a new docker layer for every `RUN`?
wouldn't it make sense to flatten those into one, and connect them with `&&`?